src/com/company/Constants.java

```diff
@@ -0,0 +1,20 @@
+package com.company;
+
+
+public class Constants {
```

Comment:
Move error messages to a separate ErrorMessages class

Reply:
done with that change